Topology Optimization/Material/Material_Interpolation.m

```diff
@@ -9,21 +9,21 @@
     end
     
     methods (Static)
-        function obj=create(ConstitutiveProperties, TypeOfMaterial, interpolation, pdim)
-            switch TypeOfMaterial
+        function obj=create(settings)
```

Comment:
How do we want to call the "settings" that enter in each class or function?

"settings"?
"input"?
"input_settings"?
"data_base"?

Reply:
I guess the proper long name for constructing a class should be "constructorParameters", but it is too long. Maybe
"constructParams"?
"cParams"?
"cP"?

Reply:
Config?

Reply:
constructorArguments?
cArg?
params?
conParams?
cSettings?

I guess cParams could be not bad, can be interpreted as configParams, constructorParams, createParams. Input is probably a bit redundant